docs/tutorial/support.md

```diff
@@ -96,13 +96,15 @@ Following platforms are supported by Electron:
 Only 64bit binaries are provided for macOS, and the minimum macOS version
 supported is macOS 10.10 (Yosemite).
 
+Electron 11.0.0 and later adds native support for Apple Silicon (`arm64`) devices.
+
 ### Windows
 
 Windows 7 and later are supported, older operating systems are not supported
 (and do not work).
 
 Both `ia32` (`x86`) and `x64` (`amd64`) binaries are provided for Windows.
-[Electron 6.0.8 and later add native support for Windows on Arm (`arm64`) devices](windows-arm.md).
```

Comment:
this was correct as-is - "Electron 6.0.8 and later" is a proxy for "versions" which as a plural subject require pluralization of the verb. To be more clear, let's instead say "include" here.

Reply:
At the risk of being pedantic... IMO this doesn't improve the grammar as "include" + "includes" have the same potential confusion as "add" and "adds". Moreover, "include" is slightly less clear than "add" in these sentences.

How about:

> Native support for Apple Silicon (`arm64`) devices was added in Electron 11.0.0.

> Native support for Windows on Arm (`arm64`) devices was added in Electron 6.0.8.